modules/tinymce/src/core/main/ts/api/util/Tools.ts

```diff
@@ -194,15 +185,17 @@ const resolve = (n, o?) => {
  * // Split a string into an array with a,b,c
  * const arr = tinymce.explode('a, b,   c');
  */
-const explode = (s, d?) => {
-  if (!s || is(s, 'array')) {
+const explode = (s: string | string[], d?: string | RegExp): string[] => {
+  if (Type.isNullable(s) || Type.isArray(s)) {
     return s;
+  } else if (s === '') {
+    return [];
+  } else {
+    return ArrUtils.map(s.split(d || ','), trim);
   }
-
-  return ArrUtils.map(s.split(d || ','), trim);
 };
```

Comment:
Another case where I reworked the logic to make it play nicer with types. Also makes sure that an empty string returns an empty array rather than an empty string.